__tests__/storage/getStopClusterByGpuid.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { Storage } from '../../src/classes/storage';
+import { fakeGroundPlacesJSON } from '../../mocks/groundPlaces';
+
+describe('getStopCluster()', () => {
+  const StorageInstance: Storage = new Storage(fakeGroundPlacesJSON);
+
+  it('should return the right StopCluster based on its Gpuid', () => {
+    const getStopClusterByGpuid = jest.spyOn(Storage.prototype, 'getStopClusterByGpuid');
+
+    StorageInstance.getStopClusterByGpuid('c|FRstrasbou@u0ts2');
+
+    expect(getStopClusterByGpuid).toHaveBeenCalledTimes(1);
+    expect(StorageInstance.getStopClusterByGpuid('c|FRstrasbou@u0ts2')).toEqual({
```

Comment:
i think you can simplify it, just leave this expect (remove the spy and the previous call)

Reply:
Done